tests/test_tokenization_roberta.py

```diff
@@ -80,12 +80,12 @@ def get_input_output_texts(self, tokenizer):
     def test_full_tokenizer(self):
         tokenizer = RobertaTokenizer(self.vocab_file, self.merges_file, **self.special_tokens_map)
         text = "lower newer"
-        bpe_tokens = ["\u0120low", "er", "\u0120", "n", "e", "w", "er"]
-        tokens = tokenizer.tokenize(text, add_prefix_space=True)
+        bpe_tokens = ["l", "o", "w", "er", "\u0120", "n", "e", "w", "er"]
```

Comment:
does this show that this PR changes the default tokenization?

Reply:
Do we still want test coverage for add_prefix_space=True?

Reply:
Dynamically adding `add_prefix_space=True` at tokenization time is not possible in fast tokenizers anymore and I think it should be deprecated in the future in slow tokenizers as well. I'll add a DeprecationWarning.